pkg/utils/log.go

```diff
@@ -75,7 +75,8 @@ func NewLogger(prefix string, level LogLevel) *Logger {
 	}
 
 	stdout, _ = os.OpenFile("/var/log/heketi/heketi.log", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
-	//stdout = os.Stdout
```

Comment:
Furthermore:

- This should not change the current behavior, but should be configurable.
- Errors from OpenFile need to be checked.